Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Cartero/Cartero](https://hosted.weblate.org/projects/cartero/cartero/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/cartero/cartero/horizontal-auto.svg)

yellowsnow3 committed:
Translated using Weblate (German)

Currently translated at 100.0% (282 of 282 strings)

Translation: Cartero/Cartero
Translate-URL: https://hosted.weblate.org/projects/cartero/cartero/de/

danirod commented:
Currently compiling Cartero 25.0 in the build farm. Too late to restart the process. These translations will have to wait for 25.1.

oltanis committed:
Translated using Weblate (Galician)

Currently translated at 93.2% (263 of 282 strings)

Translation: Cartero/Cartero
Translate-URL: https://hosted.weblate.org/projects/cartero/cartero/gl/

danirod commented:
Fine.
